src/test/java/plugins/FavoriteTest.java

```diff
@@ -0,0 +1,31 @@
+package plugins;
+
+import org.jenkinsci.test.acceptance.junit.AbstractJUnitTest;
+import org.jenkinsci.test.acceptance.junit.WithPlugins;
+import org.jenkinsci.test.acceptance.plugins.mock_security_realm.MockSecurityRealm;
+import org.jenkinsci.test.acceptance.po.FreeStyleJob;
+import org.jenkinsci.test.acceptance.po.GlobalSecurityConfig;
+import org.junit.Test;
+
+@WithPlugins({"favorite", "mock-security-realm"})
+public class FavoriteTest extends AbstractJUnitTest {
+    public static final String USER = "dev";
+
+    @Test
+    public void shouldBeAbleToFavoriteAJob() throws Exception {
+        jenkins.jobs.create(FreeStyleJob.class, "my-project");
+
+        final GlobalSecurityConfig security = new GlobalSecurityConfig(jenkins);
+        security.open();
+        MockSecurityRealm realm = security.useRealm(MockSecurityRealm.class);
+        realm.configure(USER);
+        security.save();
+
+        jenkins.login().doLogin(USER);
+
+        jenkins.open();
+        waitFor(by.id("fav_my-project")).click();
```

Comment:
Something should happen after clicking here that you can check to be sure the relevant js has correctly ejecuted

Reply:
yes. At this moment, the `div#favorites` should be empty. From my tests, the div is not even found by ATH when it's really there (could see it debuging the ATH run).

Reply:
This was resolved by the next line as we see something in the page with `icon-fav-active` CSS class.